content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -459,6 +459,56 @@ Configure profiles that users can select when creating a Terraform Integration s
    hal deploy apply
    ```
 
+### Adding authz to Named Profiles
+
+Armory recommends that you enable authorization for your Named Profiles to provide more granular control and give App Developers better guardrails. When you configure authz for Named Profiles, you need to explicitly grant permission to the role(s) you want to have access to the profile. Users who do not have permission to use a certain Named Profile do not see it as an option in Deck.
```

Comment:
> do not see it as an option in Deck.

And maybe add that the stage will fail if they try to run a stage with an unauthorized profile.